net-libs/libsignal-protocol-c/libsignal-protocol-c-9999.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
Is it pending upstream or is upstream dead? If it's dead, we should drop 9999. If it's pending upstream, we should put a link in the patch.

Reply:
Hi @thesamesam,

Sadly the upstream for this package is archived, so there's no where to send the patch to: https://github.com/signalapp/libsignal-protocol-c

I'll send a commit soon to drop the -9999 patch.

Reply:
@thesamesam I have pushed up a new commit just before the CVE fix commit that drops the -9999 ebuild.